LocalSettings.php

```diff
@@ -4256,8 +4258,118 @@
 	'wmgWikibaseRepoPropertyNamespaceID' => [
 		'default' => 862
 	],
+	'wmgWikibaseAllowLocalShortDesc' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wmgWikibaseForceLocalShortDesc' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	'wmgWikibaseEnableData' => [
+		'default' => true,
+	],
+	'wmgWikibaseEntityAccessLimit' => [
+		'default' => 400,
+		'gratisdatawiki' => 500,
+	],
+	'wmgWikibaseAllowDataAccessInUserLanguage' => [
+		'default' => false,
+		'gratisdatawiki' => true,
+	],
+	'wmgWikibaseClientNamespacesWithRepoAccess' => [
+		'default' => [],
+		'gratisdatawiki' => [
+			NS_CATEGORY,
+			NS_PROJECT,
+			NS_TEMPLATE,
+			NS_HELP,
+			828, // NS_MODULE
+		],
+	],
+	'wmgWBRepoIdGenerator' => [
+		'default' => 'mysql-upsert',
+	],
+	'wmgWBRepoIdGeneratorInErrorPingLimiter' => [
+		'default' => 0,
+		'gratisdatawiki' => 9,
+	],
+	'wmgWikibaseClientEchoIcon' => [
+		'default' => [],
+		'gratispaideiawiki' => [
+			'url' => 'https://static.miraheze.org/commonswiki/a/a4/GDechoIcon.svg',
+		],
+	],
+	'wmgEchoEnablePush' => [
+		'default' => false,
+		'gratispaideiawiki' => true,
+	],
+	// Only enable this ('wmgWikibaseClientTrackLuaFunctionCallsPerWiki') conservatively, having this on to many
```

Comment:
this comment seems to be WMF specific

Reply:
WMF specific? if I got you right, My reply would be, because I saw the comment somewhere on one example config file, _can't remember where again_

Reply:
What I mean is that we don't have Graphite, meaning that the comment isn't useful/relevant to Miraheze